be/src/Project/Services/Catalog/Catalog.Api/Services/Products/ProductsService.cs

```diff
@@ -332,6 +332,11 @@ public async Task<PagedResults<IEnumerable<ProductServiceModel>>> GetByIdsAsync(
         {
             var searchResults = await _productSearchRepository.GetAsync(model.Language, model.OrganisationId, model.Ids, model.OrderBy);
```

Comment:
**suggestion:** I would suggest adding searchTerm to the method. Why we couldn't use GetAsync method?